pkg/tagset/old/sort_uniq.go

```diff
@@ -0,0 +1,58 @@
+package tagset
+
+import "sort"
+
+// NOTE: this functionality is copied from ./pkg/util.  It will probably not be
```

Comment:
Is it duplicated because of cyclic imports?

Reply:
Worse -- it's because `./pkg/tagset` is a module, which does not depend on `./`, which is the module containing `./pkg/util`.